css/style.css

```diff
@@ -6,6 +6,153 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+
+/* navigation bar */
+
+.navbar {
+    padding: 10px 20px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-around;
+    background-color: rgb(241, 241, 241);
+    position: fixed;
+    width: 100%;
+}
+
+header a:hover {
+    color: black;
+}
+
+
+.dark {
+    color: black;
+}
+
+.logo {
+   width: 4vh;
+}
+
+.links {
+    list-style-type: none;
+    display: inline-flex;
+    flex: 3;
+    justify-content: flex-end;
+}
+
+li {
+    margin-right: 50px;
+}
+
+.navbar a {
+    text-decoration: none;
+    font-size: 1.5em;
+    color: rgb(160, 160, 160);
+}
+
+
+/* SHOWCASE */
+
+.showcase{
+    background: url(../img/first-background.jpg) no-repeat;
+    background-size: cover;
+    height: 70vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.showcase h2{
+    color: white;
+    font-size: 4em;
+    font-weight: 100;
+    margin: 0em;
+}
+
+.showcase h4 {
+    color: white;
+    font-size: 2em;
+    font-weight: 50;
+    margin: 10px 0px;
+}
+
+.button {
+    margin-top: 3vh;
+    height: 50px;
+    background-color: rgb(255, 74, 42);
+    color: white;
+    border: 0vh;
+    padding: 0px 40px;
+}
+
+/* SEGMENTS */
+.seg h2 {
+    text-align: center;
+    margin-top: 10vh;
+    font-size: 4em;
+    font-weight: 100;
+}
+
+.segments{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: baseline;
+    text-align: center;
+}
+
+.dev {
+    width: 70%;
+}
+
+.coffee {
+width: 70%;
+}
```

Comment:
Why define 2 separate classes that do the same thing here? You can create one class (for example `segment-icon`) and set the size once.
Remember, CSS classes are meant to be re-used.